cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -262,6 +266,10 @@ type LocalEtcd struct {
 	// command line except without leading dash(es).
 	ExtraArgs map[string]string
 
+	// ExtraEnvs is an extra set of envs to pass to the control plane component, e.g. GOMAXPROCS.
+	// +optional
+	ExtraEnvs []v1.EnvVar `json:"extraEnvs,omitempty"`
```

Comment:
```suggestion
	ExtraEnvs []v1.EnvVar
```

internal structs must be without json tags